R/simulations.R

```diff
@@ -0,0 +1,97 @@
+#' @title Run simulation
+#' @description Run the simulation for a given country
+#'
+#' @param countryname country name
+#' @param R0 R0
+#' @param timestep time step
+#' @param dt dt
+#' @param time_period time period
+#' @param tt_contact_matrix tt contact matrix
+#' @param newpopulation set this to NULL to use pop$n
+#' @param numberof_days number of days to run for
+#'
+#' @export
+#'
+#' @return data frame
+run_simulation <- function(countryname, R0, timestep, dt, time_period,
```

Comment:
[in scope]

however, it shouldn't include any processes or events. They're for later

Reply:
create_processes and probabilities_of_states is still out of scope

Reply:
ok, but all pertaining to states and their creation.................

Reply:
Actually create_processes will be changed signficantly in #28 and and the probabilities_of_states will change completely because there will be durations instead.

You already have created all the states in create_states, no?